pkg/hubble/filters/http.go

```diff
@@ -31,8 +31,8 @@ func httpMatchCompatibleEventFilter(types []*flowpb.EventTypeFilter) bool {
 }
 
 var (
-	httpStatusCodeFull   = regexp.MustCompile(`[1-5][0-9]{2}`)
-	httpStatusCodePrefix = regexp.MustCompile(`^([1-5][0-9]?\+)$`)
+	httpStatusCodeFull   = regexp.MustCompile(`^[1-5][0-9]{2}$`)
```

Comment:
Unrelated to your change, but these variables are named backwards >_>

Reply:
You mean `s/httpStatusCodeFull/FullHttpStatusCode/`? They are private so could be done on the way.

Reply:
I misread the regexes. It doesn't need to be swapped.